### ISSUE TITLE
[LMS] <TC_LEARNER_50>: <Dashboard search isn't working>

### ISSUE THREAD

**Issue by fayyazahmed66** [openedx/wg-build-test-release#321]
### Release

quince

### Expected behavior

The search should work properly.

### Actual behavior

The search isn't working according to expectations.

### Steps to reproduce

1. Go to the LMS dashboard
2. type any already existing course in the search
3. click on search button 

### Additional information

<img width="1679" alt="dashboard search" src="https://github.com/openedx/wg-build-test-release/assets/55684124/00eee850-12fc-4fe4-af96-2c85d939ea95">

**Comment by mariajgrimaldi** [comment]
I'll be researching this in the next few days

**Comment by mariajgrimaldi** [comment]
@regisb: can we check that the demo server search configurations are correct? I'm trying to configure the course search in my local environment, but I'm having a bit of trouble, I'm following this post to do it: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/edx_search.html

**Comment by regisb** [comment]
:thinking: I tried to search for a course on the demo server: https://quince.demo.edly.io/courses A search for "demo" yields no result, but looking for "demonstration" does return one result.

![image](https://github.com/openedx/wg-build-test-release/assets/44319/a6df73b5-84c5-4cb0-9350-b70fc1c087a2)

Was the issue fixed since it was reported?

**Comment by mariajgrimaldi** [comment]
We didn't classify this issue as a release blocker since the learner dashboard MFE is now the default dashboard, but I'll let the maintainers know that we've encountered it. I'll try to reproduce it with my local environment in the meantime.

**Comment by 0x29a** [comment]
@mariajgrimaldi, were you able to reproduce this issue locally?

**Comment by mariajgrimaldi** [comment]
Hi @0x29a, I went on vacation shortly after I published the last comment. I'll try reproducing the issue locally this week or next. Thanks for the patience!

**Comment by mariajgrimaldi** [comment]
Hi there @0x29a, thanks for your patience. I followed these instructions and made it work in my Tutor nightly environment:
- Configure my search engine as `SEARCH_ENGINE = "search.elastic.ElasticSearchEngine"` for the LMS/CMS
- Ensure my courses had an enrollment date in the past
- Reindex all courses by running `./manage.py cms reindex_course --all`
- Considering [what gets indexed](https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/edx_search.html#which-data-gets-indexed), I searched for some subsections:
![image](https://github.com/openedx/wg-build-test-release/assets/64440265/04e2433c-d11e-4f0a-bc03-01ea800a72d0)
To be sure this was working in quince, I also tested it in a remote quince environment we have running:
![image](https://github.com/openedx/wg-build-test-release/assets/64440265/7605d1e9-0157-4688-bf05-5b6bcbf2b9a7)
 Since this is working, I'll be closing this issue. If something else is found, then we can open a new one.
